.changes/next-release/service.lookoutequipment-major-1617910916305435000.json

```diff
@@ -0,0 +1,9 @@
+{
+ "ID": "service.lookoutequipment-major-1617910916305435000",
+ "SchemaVersion": 1,
+ "Module": "service/lookoutequipment",
+ "Type": "major",
+ "Description": "new service client",
+ "MinVersion": "",
+ "AffectedModules": null
```

Comment:
I think this `MinVersion` needs to be v1.0.0 or the release tool won't look at `type` correctly. same for mgn model.

Reply:
updated